src/mainboard/novacustom/mtl-h/variants/igpu/ramstage.c

```diff
@@ -1,8 +1,19 @@
-/* SPDX-License-Identifier: CC-PDDC */
+/* SPDX-License-Identifier: GPL-2.0-only */
 
-/* Please update the license if adding licensable material. */
+#include <smbios.h>
 
-/*
- * This is a placeholder, if there ever is something variant specific to
- * add to ramstage, do it here.
- */
+void smbios_fill_dimm_locator(const struct dimm_info *dimm,
+	struct smbios_type17 *t)
+{
+	switch (dimm->ctrlr_num) {
+	case 0:
+		t->device_locator = smbios_add_string(t->eos, "RAM2");
```

Comment:
Is it really reversed on iGPU vs dGPU model?

Reply:
yes, DIMMA is RAM2, DIMMB is RAM1